web-api/src/persistence/elasticsearch/getDocketNumbersByStatusAndByJudge.test.ts

```diff
@@ -17,14 +17,18 @@ describe('getDocketNumbersByStatusAndByJudge', () => {
     statuses: CAV_AND_SUBMITTED_CASE_STATUS,
   };
 
-  const responseResults = [
-    { docketNumber: '11315-18' },
-    { docketNumber: { S: '11316-18' } },
-  ];
+  const standaloneCase = { docketNumber: '11315-18' };
+  const leadCase = { docketNumber: '11316-18', leadDocketNumber: '11316-18' };
+  const memberCase = { docketNumber: '11317-18', leadDocketNumber: '11316-18' };
+  const responseResults = [standaloneCase, leadCase, memberCase];
```

Comment:
👏🏿 👏🏿 
Separating the docket numbers into prescriptive variable names really do help.